package.json

```diff
@@ -25,7 +25,7 @@
     },
     "dependencies": {
         "@chili-publish/grafx-shared-components": "^0.72.8",
-        "@chili-publish/studio-sdk": "^1.13.0",
+        "@chili-publish/studio-sdk": "^1.14.0-rc.1",
```

Comment:
can we use latest released version as long as you don't need to use anything specific from the rc SDK?